cmd/skopeo/list_tags.go

```diff
@@ -39,12 +40,12 @@ func tagsCmd(global *globalOptions) *cobra.Command {
 		retryOpts: retryOpts,
 	}
 	cmd := &cobra.Command{
-		Use:   "list-tags [command options] REPOSITORY-NAME",
-		Short: "List tags in the transport/repository specified by the REPOSITORY-NAME",
-		Long: `Return the list of tags from the transport/repository "REPOSITORY-NAME"
+		Use:   "list-tags [command options] transport",
```

Comment:
How about `SOURCE-IMAGE`?  `transport` is strictly speaking not accurate as we need `transport+image`.

Reply:
Well, “image” also seems quite inappropriate, when the input is some kind of aggregate (repo/archive) and the output is a _list_ of images.

Reply:
> How about `SOURCE-IMAGE`? `transport` is strictly speaking not accurate as we need `transport+image`.

There doesn't seem to be a suitable word to describe it, what do you think about it @vrothberg

Reply:
LGTM